model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py

```diff
@@ -2323,6 +2344,12 @@ async def execute(
                 "Only code_evaluator tool is supported for batch completions."
             )
 
+        additional_engine_args = infer_addition_engine_args_from_model_name(
+            request.model_config.model
+        )
+        if additional_engine_args.gpu_memory_utilization is not None:
+            request.max_gpu_memory_utilization = additional_engine_args.gpu_memory_utilization
```

Comment:
instead of putting this in request, can you update the bundle config?

Reply:
It looks like the bundle isn't actually being used outside the context of this request, so it seems simpler to have a separate DTO for the Gateway vs the engine.